docs/application/native/guides/applications/overview.md

```diff
@@ -44,6 +44,10 @@ When your application becomes visible again, the `app_resume_cb()` callback is i
 - All applications on top of your application in the window stack finish.
 - An alarm is triggered for your application, bringing it to the front and hiding other applications.
 
+  > **Note**
```

Comment:
Suggestion:
The **Note** is for `app_resume_cb()` and `app_pause_cb()`. The **Note** should also be general. Here, it is aligned with the bullet list.
The alignment should be:

![line47](https://user-images.githubusercontent.com/36875286/52618003-7fc2fa00-2e95-11e9-967f-ba0fc9df301f.JPG)

Please verify and fix the alignment.